src/test/java/model/PointHandlerTest.java

```diff
@@ -8,7 +8,7 @@
 class PointHandlerTest {
     private final PointHandler pointHandler = new PointHandler();
 
-    @Test
```

Comment:
better remove all the tests . It causes more unwelcome questions from your teacher / lector.